ui/oopsyraidsy/oopsyraidsy.ts

```diff
@@ -1,41 +1,46 @@
+import { UnreachableCode } from '../../resources/not_reached';
 import { callOverlayHandler, addOverlayListener } from '../../resources/overlay_plugin_api';
 import UserConfig from '../../resources/user_config';
 
-import { OopsySummaryList } from './oopsy_summary_list';
-import { OopsyLiveList } from './oopsy_live_list';
 import { DamageTracker } from './damage_tracker';
+import oopsyFileData from './data/oopsy_manifest.txt';
 import { MistakeCollector } from './mistake_collector';
+import { OopsyListView } from './oopsy_list_view';
+import { OopsyLiveList } from './oopsy_live_list';
 import defaultOptions from './oopsy_options';
+import { OopsySummaryList } from './oopsy_summary_list';
 
 import './oopsyraidsy_config';
 
-import oopsyFileData from './data/oopsy_manifest.txt';
-
 import '../../resources/defaults.css';
 import './oopsy_common.css';
 
 UserConfig.getUserConfigLocation('oopsyraidsy', defaultOptions, () => {
   const options = { ...defaultOptions };
-  let listView;
-  let mistakeCollector;
+  let initListView: OopsyListView | undefined;
```

Comment:
This feels weird, is the `| undefined` really necessary if it's a let?

Reply:
Oh you're right, thanks!